src/components/templates/Camp/components/Map.tsx

```diff
@@ -42,22 +52,43 @@ export const Map: React.FC<PropsType> = ({
               width: room.width_percent + "%",
               height: room.height_percent + "%",
             }}
-            key={idx}
-            onClick={() => openRoomModal(room)}
+            key={room.title}
+            onClick={() => {
+              setClicked((prev) => !prev);
```

Comment:
do we need clicked/setClicked? Can it not be inferred from whether `roomClicked === ""`?

Reply:
no because roomClicked states which room has last been clicked (in order to fix that problem of having the pop-up rendering over the room instead of below).

clicked specifies whether the the small or large pop-up should be displayed when hovering over it.